:sparkles: add multi receipts & Invoice Splitter auto extraction (#83)
mindee/mindee-api-php: README.md

echo strval($apiResponse->document);
```

## Full PDF support

> Some features such as Invoice Splitter auto-extraction & Multi Receipts auto-extraction require the library to use, [ImageMagick](https://www.php.net/manual/en/imagick.setup.php) which in turn uses [GhostScript](https://www.ghostscript.com/).

Reviewer: ```suggestion
Some features such as Invoice Splitter auto-extraction & Multi Receipts auto-extraction require the [ImageMagick](https://www.php.net/manual/en/imagick.setup.php) library,  which in turn requires [GhostScript](https://www.ghostscript.com/).
```

### Unix

> ImageMagick is usually bundled with most installation, but you can install it based on your distro. More details [here](https://imagemagick.org/script/advanced-linux-installation.php).

Reviewer: ```suggestion
ImageMagick is usually bundled with most installations, if not you can install it using your distribution's package manager.
More details [here](https://imagemagick.org/script/advanced-linux-installation.php).
```

> Ghostscript can be installed from the [website download page](https://ghostscript.com/releases/gsdnld.html), or from the apt store:

Reviewer: ```suggestion
Ghostscript can be installed from the [website download page](https://ghostscript.com/releases/gsdnld.html), or if using an apt based distribution:
```

```bash
sudo apt-get update
sudo apt-get install -y ghostscript
```

In some cases, you might also want to authorize the ImageMagick policy to edit PDF files:

```bash
DQT='"'
SRC="rights=${DQT}none${DQT} pattern=${DQT}PDF${DQT}"
RPL="rights=${DQT}read|write${DQT} pattern=${DQT}PDF${DQT}"
sudo sed -i "s/$SRC/$RPL/" /etc/ImageMagick-6/policy.xml
```

### MacOS

Brew will be required for this install.

```bash
brew update
brew install imagemagick
pecl install imagick # Might not be needed in some instances. Do not try to install before installing PHP & Pecl.
```

### Windows
You can install [Ghostscript](https://ghostscript.com/releases/gsdnld.html) by downloading it, or simply by using [Chocolatey](https://chocolatey.org/).

```
choco install ghostscript --version 10.03.1 -y
```

**⚠️ Important note if you are using Windows** 

> The `gs` alias might not be available by default, but it is possible to bind it fairly simply by either adding gswin32c.exe or gswin64c.exe to $PATH and then adding a symlink in powershell using:

Reviewer: ```suggestion
The `gs` alias might not be available by default, but it is possible to bind it fairly simply by either adding `gswin32c.exe` or `gswin64c.exe` to your `$PATH` and then adding a symlink in powershell using:
```
